.github/workflows/ci.yml

```diff
@@ -314,7 +409,364 @@ jobs:
           fi
 
       - name: Run GoogleTests
+        shell: bash
         env:
           GTEST_TOTAL_SHARDS: 2
           GTEST_SHARD_INDEX: ${{ matrix.shard_index }}
         run: ${{ format('./zcash-gtest{0}', matrix.file_ext) }}
+
+  # Not designed for Windows # (disabled to save costs)
+  univalue:
+    name: univalue tier ${{ matrix.tier }} platform ${{ matrix.platform }}
+    needs: [setup, build]
+    runs-on: ${{ matrix.test_os }}
+    continue-on-error: ${{ matrix.tier != 1 }}
+    strategy:
+      matrix:
+        include: ${{ fromJson(needs.setup.outputs.test_matrix) }}
+    steps:
+      - uses: actions/checkout@v4
+        if: runner.os != 'Windows' # (disabled to save costs)
```

Comment:
Rather than disabling every individual step, we should have a separate `test_matrix` subset (maybe named `unix_test_matrix`) that excludes Windows.

Reply:
Yes, that could be a good workaround, thus avoiding several if statements